crates/runtime/benches/snapshots/bench__s3_sqlite_file_tpcds_q75.snap

```diff
@@ -5,5 +5,5 @@ expression: records_pretty
 +-----------+------+------------+------------+---------------+---------------+-------------+-------------+----------------+-------------------------+
 | prev_year | year | i_brand_id | i_class_id | i_category_id | i_manufact_id | prev_yr_cnt | curr_yr_cnt | sales_cnt_diff | sales_amt_diff          |
 +-----------+------+------------+------------+---------------+---------------+-------------+-------------+----------------+-------------------------+
-| 1999      | 2000 | 4003001    | 3          | 4             | 376           | 6362        | 4853        | -1509          | -120879.960000000000000 |
+| 1999      | 2000 | 4003001    | 3          | 4             | 376           | 6362        | 4853        | -1509          | -120879.960000000016384 |
```

Comment:
Double checked that original query returns the same result when running unmodified in SQLite